cmd/fortifyExecuteScan.go

```diff
@@ -0,0 +1,647 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"math"
+	"os"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/v31/github"
+	"github.com/google/uuid"
+
+	"github.com/piper-validation/fortify-client-go/models"
+
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/fortify"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/maven"
+	"github.com/SAP/jenkins-library/pkg/piperutils"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+
+	piperGithub "github.com/SAP/jenkins-library/pkg/github"
+)
+
+var auditStatus map[string]string
+
+const checkString = "<---CHECK FORTIFY---"
+const classpathFileName = "cp.txt"
+
+func fortifyExecuteScan(config fortifyExecuteScanOptions, telemetryData *telemetry.CustomData, influx *fortifyExecuteScanInflux) error {
+	auditStatus = map[string]string{}
+	sys := fortify.NewSystemInstance(config.ServerURL, config.APIEndpoint, config.AuthToken, time.Second*30)
+	c := command.Command{}
+	// reroute command output to loging framework
+	c.Stdout(log.Entry().Writer())
+	c.Stderr(log.Entry().Writer())
+	c.SetEnv(os.Environ())
```

Comment:
This should be unnecessary. See [appendEnvironment](https://github.com/SAP/jenkins-library/blob/master/pkg/command/command.go#L95).